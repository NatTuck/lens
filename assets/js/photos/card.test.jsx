import React from 'react';
import { render } from '@testing-library/react';
import RTR from 'react-test-renderer';

import PhotoCard from './card';

test('photo card shows desc', () => {
  const photo  = { data: "about:blank", desc: "no photo" };
  const result = render(<PhotoCard photo={photo} />);
  expect(result.queryByText("no photo")).toBeTruthy();
});

test('render photo card snapshot', () => {
  const photo  = { data: "about:blank", desc: "no photo" };
  const result = RTR.create(<PhotoCard photo={photo} />);
  expect(result).toMatchSnapshot();
});

