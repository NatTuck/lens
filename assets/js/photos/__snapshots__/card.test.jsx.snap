// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render photo card snapshot 1`] = `
<div
  className="col-4 card"
>
  <img
    className="card-img"
    src="about:blank"
  />
  <p
    className="card-text"
  >
    no photo
  </p>
</div>
`;
